code/modules/mob/living/carbon/brain/life.dm

```diff
@@ -28,13 +28,12 @@
 		var/discomfort = min( abs(exposed_temperature - bodytemperature)*(exposed_intensity)/2000000, 1.0)
 		adjustFireLoss(5.0*discomfort)
 
-/mob/living/carbon/brain/handle_regular_status_updates()
+/mob/living/carbon/brain/Life()
```

Comment:
This is another thing to refactor eventually.

brains have no need to be carbons when they can do just fine as `/mob/living/brain`.

Them being carbons has SOOOO many whacky behaviors that aren't good.

Reply:
This looks more suited for moving to `update_stat` - particularly the aspect that checks for `container`